application/src/test/java/com/sanction/lightning/resources/FacebookResourceTest.java

```diff
@@ -172,14 +172,28 @@ public void testGetMediaBytes() {
   /* Publish Tests */
   @Test
   public void testPublishWithNullUsername() {
-    Response response = resource.publish(key, null, null, null, null);
+    Response response = resource.publish(key, null, null, null, null, null, null);
+
+    assertEquals(response.getStatusInfo(), Response.Status.BAD_REQUEST);
+  }
+
+  @Test
+  public void testPublishWithNullType() {
+    Response response = resource.publish(key, null, null, null, null, null, null);
+
+    assertEquals(response.getStatusInfo(), Response.Status.BAD_REQUEST);
+  }
+
+  @Test
+  public void testPublishWithBadTypeValue() {
+    Response response = resource.publish(key, null, null, null, "Test", null, null);
 
     assertEquals(response.getStatusInfo(), Response.Status.BAD_REQUEST);
```

Comment:
Similar thing on this test. I think it will return a `BAD_REQUEST` when it hits the null username, which isn't testing the bad type value.